app/assets/stylesheets/helpers/_header.scss

```
@@ -446,19 +447,24 @@
   }
 
   .survey-form-button {
-    @include button(govuk-colour("white"));
     @include bold-19($line-height: 1.1);
+    padding: 10px 15px 5px 15px;
```

Possibly use the `govuk-spacing()` function here?

```suggestion
    padding: govuk-spacing(2) govuk-spacing(3) govuk-spacing(1) govuk-spacing(3);
```

---

I'm slightly hesitant to bother changing this one as the rest of the file is riddled with `px` values, many of which would match the spacing scale. Am I a bad person?

---

I'd change it and I'm a bad person. But mainly because we've got to start somewhere with updating the bespoke styles.

---

Changed. Can I get a re-review please?